### DISCUSSION TITLE
Rename and Relocate

### DISCUSSION THREAD

**Discussion by reyemraphi** [advplyr/audiobookshelf#1960] [Ideas]
Would it be possible to implement a tool (e.g. merge to m4b, embed meta tags) to automatically rename and move a downloaded file/folder to a specific format, like mp3tag in this guide? (https://github.com/seanap/Plex-Audiobook-Guide#configure-mp3tag)

**Comment by kenniy** [comment]
Seems similar to whats implemented in [this PR](https://github.com/advplyr/audiobookshelf/pull/1521). Been 'Open' for last 6 months though, with some conflicts to be resolved.
